first_order_model/config/paper_configs/personalization/netadapt.yaml

```diff
@@ -0,0 +1,111 @@
+dataset_params:
```

Comment:
please move to a new folder under configs (because this has no relevance to personalization), have a clear description of what this is doing, and what its relevance relative to other similar folders are, and get rid of any unnecessary files.